presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -590,10 +592,32 @@ private void testCreateTableAs(Session session, HiveStorageFormat storageFormat)
     @Test
     public void testCreatePartitionedTableAs()
     {
-        testWithAllStorageFormats(this::testCreatePartitionedTableAs);
+        testWithAllStorageFormats(this::createPartitionedTableAs);
     }
 
-    private void testCreatePartitionedTableAs(Session session, HiveStorageFormat storageFormat)
+    @Test
+    public void testPropertiesTable()
+    {
+        @Language("SQL") String createTable = "" +
+                "CREATE TABLE test_show_properties" +
+                " WITH (" +
+                "format = 'orc', " +
+                "partitioned_by = ARRAY[ 'SHIP_PRIORITY', 'ORDER_STATUS' ]," +
```

Comment:
Nit: don't put spaces around array brackets. Also, use lowercase for the column names.
```sql
partitioned_by = ARRAY['ship_priority', 'order_status'],
orc_bloom_filter_columns = ARRAY['ship_priority', 'order_status'],